src/DurableTask.SqlServer.AzureFunctions/SqlDurabilityProviderStartup.cs

```diff
@@ -1,19 +1,21 @@
-﻿// Copyright (c) Microsoft Corporation.
+// Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
 // Reference: https://docs.microsoft.com/en-us/azure/azure-functions/functions-dotnet-dependency-injection
```

Comment:
Let's remove this comment now that it's no longer relevant.

Reply:
I meant the comment on line 4. Not sure why GitHub is highlighting line 1.

Reply:
Removed the comment. Yes, even I am confused about line1.